Crash when trying to switch to my currently playing playlist

This is a super-frequent crash that happens when I am playing a video on popup (that I have started from the What's New tab) and I tap on the Notification in order to get to the playlist I am currently watching. I don't mean a normal playlist, I mean the videos I have been enqeuing and watching.

```
{
   "user_action": "ui error",
   "exceptions": [
      "android.content.res.Resources$NotFoundException: File res\/drawable\/ic_volume_up_white_24dp.xml from drawable resource ID #0x7f08015c. If the resource you are trying to use is a vector resource, you may be referencing it in an unsupported way. See AppCompatDelegate.setCompatVectorFromResourcesEnabled() for more info.\n\tat android.content.res.Resources.loadDrawable(Resources.java:2103)\n\tat android.content.res.Resources.getDrawable(Resources.java:700)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.getThemedDrawable(ServicePlayerActivity.java:706)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.onMaybeMuteChanged(ServicePlayerActivity.java:701)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.onCreateOptionsMenu(ServicePlayerActivity.java:156)\n\tat android.app.Activity.onCreatePanelMenu(Activity.java:2538)\n\tat androidx.fragment.app.FragmentActivity.onCreatePanelMenu(FragmentActivity.java:325)\n\tat androidx.appcompat.view.WindowCallbackWrapper.onCreatePanelMenu(WindowCallbackWrapper.java:94)\n\tat androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.onCreatePanelMenu(AppCompatDelegateImpl.java:2830)\n\tat androidx.appcompat.view.WindowCallbackWrapper.onCreatePanelMenu(WindowCallbackWrapper.java:94)\n\tat androidx.appcompat.app.ToolbarActionBar.populateOptionsMenu(ToolbarActionBar.java:455)\n\tat androidx.appcompat.app.ToolbarActionBar$1.run(ToolbarActionBar.java:56)\n\tat android.os.Handler.handleCallback(Handler.java:733)\n\tat android.os.Handler.dispatchMessage(Handler.java:95)\n\tat android.os.Looper.loop(Looper.java:136)\n\tat android.app.ActivityThread.main(ActivityThread.java:5133)\n\tat java.lang.reflect.Method.invokeNative(Native Method)\n\tat java.lang.reflect.Method.invoke(Method.java:515)\n\tat com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)\n\tat com.android.internal.os.ZygoteInit.main(ZygoteInit.java:602)\n\tat dalvik.system.NativeStart.main(Native Method)\nCaused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #1: invalid drawable tag vector\n\tat android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:933)\n\tat android.graphics.drawable.Drawable.createFromXml(Drawable.java:877)\n\tat android.content.res.Resources.loadDrawable(Resources.java:2099)\n\t... 20 more\norg.xmlpull.v1.XmlPullParserException: Binary XML file line #1: invalid drawable tag vector\n\tat android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:933)\n\tat android.graphics.drawable.Drawable.createFromXml(Drawable.java:877)\n\tat android.content.res.Resources.loadDrawable(Resources.java:2099)\n\tat android.content.res.Resources.getDrawable(Resources.java:700)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.getThemedDrawable(ServicePlayerActivity.java:706)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.onMaybeMuteChanged(ServicePlayerActivity.java:701)\n\tat org.schabi.newpipe.player.ServicePlayerActivity.onCreateOptionsMenu(ServicePlayerActivity.java:156)\n\tat android.app.Activity.onCreatePanelMenu(Activity.java:2538)\n\tat androidx.fragment.app.FragmentActivity.onCreatePanelMenu(FragmentActivity.java:325)\n\tat androidx.appcompat.view.WindowCallbackWrapper.onCreatePanelMenu(WindowCallbackWrapper.java:94)\n\tat androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.onCreatePanelMenu(AppCompatDelegateImpl.java:2830)\n\tat androidx.appcompat.view.WindowCallbackWrapper.onCreatePanelMenu(WindowCallbackWrapper.java:94)\n\tat androidx.appcompat.app.ToolbarActionBar.populateOptionsMenu(ToolbarActionBar.java:455)\n\tat androidx.appcompat.app.ToolbarActionBar$1.run(ToolbarActionBar.java:56)\n\tat android.os.Handler.handleCallback(Handler.java:733)\n\tat android.os.Handler.dispatchMessage(Handler.java:95)\n\tat android.os.Looper.loop(Looper.java:136)\n\tat android.app.ActivityThread.main(ActivityThread.java:5133)\n\tat java.lang.reflect.Method.invokeNative(Native Method)\n\tat java.lang.reflect.Method.invoke(Method.java:515)\n\tat com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)\n\tat com.android.internal.os.ZygoteInit.main(ZygoteInit.java:602)\n\tat dalvik.system.NativeStart.main(Native Method)\n"
   ],
   "os": "Linux Android 4.4.4 - 19",
   "time": "2020-03-31 22:33",
   "request": "App crash, UI failure",
   "package": "org.schabi.newpipe",
   "service": "none",
   "content_language": "GB",
   "user_comment": "",
   "version": "0.19.1"
}
```

What causes this error? And why are there backslashes and forward slashes there in that filename?

--- a/app/src/main/java/org/schabi/newpipe/player/ServicePlayerActivity.java
+++ b/app/src/main/java/org/schabi/newpipe/player/ServicePlayerActivity.java
@@ -3,17 +3,9 @@ package org.schabi.newpipe.player;
 import android.content.ComponentName;
 import android.content.Intent;
 import android.content.ServiceConnection;
-import android.graphics.drawable.Drawable;
 import android.os.Bundle;
 import android.os.IBinder;
 import android.provider.Settings;
-
-import androidx.appcompat.app.AppCompatActivity;
-import androidx.recyclerview.widget.LinearLayoutManager;
-import androidx.recyclerview.widget.RecyclerView;
-import androidx.appcompat.widget.Toolbar;
-import androidx.recyclerview.widget.ItemTouchHelper;
-
 import android.util.Log;
 import android.view.Menu;
 import android.view.MenuItem;
@@ -25,6 +17,12 @@ import android.widget.PopupMenu;
 import android.widget.ProgressBar;
 import android.widget.SeekBar;
 import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.recyclerview.widget.ItemTouchHelper;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
 import com.google.android.exoplayer2.PlaybackParameters;
 import com.google.android.exoplayer2.Player;
@@ -698,13 +696,10 @@ public abstract class ServicePlayerActivity extends AppCompatActivity
             item.setTitle(player.isMuted() ? R.string.unmute : R.string.mute);
 
             //2) Icon change accordingly to current App Theme
-            item.setIcon(player.isMuted() ? getThemedDrawable(R.attr.volume_off) : getThemedDrawable(R.attr.volume_on));
+            // using rootView.getContext() because getApplicationContext() didn't work
+            item.setIcon(player.isMuted()
+                    ? ThemeHelper.resolveResourceIdFromAttr(rootView.getContext(), R.attr.volume_off)
+                    : ThemeHelper.resolveResourceIdFromAttr(rootView.getContext(), R.attr.volume_on));
         }
-    }
-
-    private Drawable getThemedDrawable(int attribute) {
-        return getResources().getDrawable(
-                getTheme().obtainStyledAttributes(R.style.Theme_AppCompat, new int[]{attribute})
-                        .getResourceId(0, 0));
     }
 }

